### PR TITLE
Stubs Snapshot logic

### PR DESCRIPTION
**Pull request by GantMan**
### If a `.solidarity` file is found
image

### No `.solidarity` file
image

### No `.solidarity` file -> List plugins
image

### No `.solidarity` file -> No plugins found
image

Plugin: https://github.com/infinitered/solidarity-react-native

### CODE REVIEW

File: src/extensions/solidarity-extension.js [hunk 1]
```diff
@@ -7,4 +7,9 @@ module.exports = (context) => {
     getSolidaritySettings: require('./functions/getSolidaritySettings'),
     skipRule: require('./functions/skipRule')
   }
+  // place for plugins
+  context.pluginsList = []
+  // Flavored separator
+  context.printSeparator = () =>
+    context.print.info(context.print.colors.america('-----------------------------------------------------------------------------------'))
```
Comment (skellock) [inline src/extensions/solidarity-extension.js]: Here we come to save the mf day ya!

File: src/commands/snapshot.js [hunk 2]
```diff
@@ -1,7 +1,72 @@
+const { propEq, filter, head } = require('ramda')
+const NONE = 'None'
+const DO_NOTHING = 'Nothing to do ¯\\_(ツ)_/¯'
```
Comment (skellock) [inline src/commands/snapshot.js]: Mr Shruggy! You’re alive!

File: src/commands/snapshot.js [hunk 3]
```diff
@@ -1,7 +1,72 @@
...
+
+const createSolidarityFile = async (context) => {
+  const { print, printSeparator } = context
+  // list visible plugins
+  printSeparator()
+  print.info('Available technology plugins:\n')
+  if (context.pluginsList.length > 0) {
+    const pluginOptions = [NONE]
+    context.pluginsList.map((plugin) => {
+      print.info(`   ${plugin.name}:\t ${plugin.description}`)
```
Comment (skellock) [inline src/commands/snapshot.js]: Sadly this tab won’t align things that have > 4 character difference. Or at least that’s how I understand tabs to work in the terminal.

Comment (GantMan) [reply src/commands/snapshot.js]: Damn!  I was trying to use table print, but it needed such a specific structure.  I'll get back to formatting on the final layer of polish.

**Pull request merged**